spec/factories/bundle_instance.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :bundle_instance do
+    bundle { FactoryGirl.create(:application_bundle).bundle }
```

Comment:
Because I want to generate an application_instance as well as a bundle. Is there a better way to do that?